tests/hazmat/primitives/test_dh.py

```diff
@@ -16,10 +16,16 @@
 from cryptography.hazmat.primitives.asymmetric import dh
 from cryptography.utils import int_from_bytes
 
+from .fixtures_dh import FFDH3072_P
 from ...doubles import DummyKeySerializationEncryption
 from ...utils import load_nist_vectors, load_vectors_from_file
 
 
+@pytest.fixture()
```

Comment:
I'm not a big fan of this fixtures style. Is there a reason not to just call a function in the test?